CharacterCreation/04_03_SMGs_&_Shotguns.txt

```diff
@@ -209,57 +267,90 @@ Auto Fire Rate: 	2
 Reload DC:		45
 Fire Modes:		Pump Action
 Armor Piercing Level:	2	
 Reflex Modifier:	+5	
 Movement Speed Penalty:	-0.5m
-Notes/Description: Slug shells do 125 damage for all range brackets
-		
-----Partix LA10 B-CQ	$2029	----
-Laser Damage, Ranged Miss Chance:
-	110 damage with a miss chance of 2 out to 10m	
-	 90 damage with a miss chance of 3 out to 20m	
-	 70 damage with a miss chance of 3 out to 40m	
+
+== Partix LA10 B-CQ	$2029	==
+
++---------+--------------+----------+
+| Damage  |  Miss Chance |  Out to  |
++---------+--------------+----------+
+|   110   |      2       |    10m   |   
++---------+--------------+----------+
+|   90    |      3       |    20m   |
++---------+--------------+----------+
+|   70    |      3       |    40m   |
++---------+--------------+----------+
+	
 Mag Size:		5 rounds of 15 kWh, mag-fed. $6 per magazine
 Reload DC:		35
 Fire Modes:		Pump Action
 Armor Piercing Level:	n/a	
 Reflex Modifier:	-5	
 Movement Speed Penalty:	-0.5m
-Notes/Description: The Partix LA10 deals different amounts of damage 
-	at 3 distinct distances instead of the norm (2 distances).
-	Mag-fed, 2 Actions to reload.
+
+	The Partix LA10 deals different amounts of damage at 3 distinct distances 
+instead of the norm (2 distances). Mag-fed, 2 Actions to reload. All hits deal
+laser damage.
 	
-----NFA Combat 7 Shotgun	$2691	----
-Ballistic Damage, Ranged Miss Chance:		
-	135 damage with a miss chance of 2 out to 10m	
-	100 damage with a miss chance of 3 out to 20m	
-	110 damage with a miss chance of 4 out to 50m - Slug Only 
+== NFA Combat 7 Shotgun	$2691	==
+
++---------+--------------+----------+
+| Damage  |  Miss Chance |  Out to  |
++---------+--------------+----------+
+|   135   |      2       |    10m   |   
++---------+--------------+----------+
+|   100   |      3       |    20m   |
++---------+--------------+----------+
+|   110   |4 (Slug only) |    50m   |
++---------+--------------+----------+
+
+	Slug shells do 110 damage for all range brackets
+
 Mag Size:		7 rounds of 12-gauge, mag-fed. $9 per magazine
 Reload DC:		30
 Fire Modes:		Semi
 Armor Piercing Level:	1	
 Reflex Modifier:	+6	
 Suppression Level:	Medium	
-Notes/Description: Slugs do 110 damage for all range brackets.
+
 	Mag-fed, 2 Actions to reload.
 
 ===== Level 10 Weapons =====
 
-----Matthews Model 3x12	$1492	----
-Ballistic Damage, Ranged Miss Chance:		
-	140 damage with a miss chance of 3 out to 10m	
-	105 damage with a miss chance of 4 out to 20m	
-	115 damage with a miss chance of 5 out to 50m - Slug Only 
+== Matthews Model 3x12	$1492	==
+
++---------+--------------+----------+
+| Damage  |  Miss Chance |  Out to  |
++---------+--------------+----------+
+|   140   |      3       |    10m   |   
++---------+--------------+----------+
+|   105   |      4       |    20m   |
++---------+--------------+----------+
+|   115   |5 (Slug only) |    50m   |
++---------+--------------+----------+
+
+	Slug shells do 120 damage for all range brackets

```

Comment:
Deleted line 271 says slug damage is 115.

Reply:
very good catch. Good eye.